network-api/networkapi/settings.py

```diff
@@ -462,30 +462,42 @@
 # LOGGING
 LOGGING = {
     'version': 1,
-    'disable_existing_loggers': False,
+    'disable_existing_loggers': True,
+    'filters': {
+        'require_debug_true': {
+            '()': 'django.utils.log.RequireDebugTrue',
+        }
+    },
     'formatters': {
-        'simple': {
-            'format': '[%(asctime)s] [%(levelname)s] %(message)s'
-        },
+        'verbose': {
+            'format': '%(asctime)s [%(levelname)s] %(message)s',
+            'datefmt': '%Y-%m-%d %H:%M:%S'
+        }
     },
     'handlers': {
-        'console': {
+        'debug': {
+            'level': 'DEBUG',
             'class': 'logging.StreamHandler',
-            'formatter': 'simple',
+            'filters': ['require_debug_true'],
+            'formatter': 'verbose'
         },
+        'error': {
+            'level': 'ERROR',
+            'class': 'logging.StreamHandler',
+            'filters': ['require_debug_true']
```

Comment:
Do we really only wan to see errors when debug is on? This feels like a log level that should kick in regardless of debug value.

Reply:
Same as Pomax: I would prefer to see errors messages too on prod.